src/index.spec.ts

```diff
@@ -0,0 +1,323 @@
+import expect from "expect";
+
+import calculateDepreciation, {
+  formatPercentage,
```

Comment:
It does not exist anymore?
```suggestion
```

Reply:
thanks, it's removed https://github.com/kontist/asset-depreciation-calculator/pull/1/commits/7b4834fa0852641cb8978ab4ff7c799ea44d73da